### PR TITLE
refactor: renamed slice methods and used stream request in cypress test

### PR DESCRIPTION
**Pull request by jimezesinachi**
Signed-off-by: Jim Ezesinachi <jim@animo.id>

### CODE REVIEW

File: client/src/pages/onboarding/steps/AcceptCredential.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export const AcceptCredential: React.FC<Props> = ({ content, connectionId, crede
     CredentialEventTypes.CredentialStateChanged,
     (event: { payload: { credentialRecord: CredentialExchangeRecord } }) => {
       if (event.payload.credentialRecord.connectionId === connectionId) {
-        dispatch(fetchCredentialEventByConnectionId(event.payload.credentialRecord))
+        dispatch(setCredential(event.payload.credentialRecord))
```
Comment (janrtvld) [inline client/src/pages/onboarding/steps/AcceptCredential.tsx]: I think its more `AddCredential`, as you're adding it to a list of credentials here.

Comment (jimezesinachi) [reply client/src/pages/onboarding/steps/AcceptCredential.tsx]: Got it. On it.

**Review by janrtvld — APPROVED** [review]

**Comment by janrtvld** [comment]
All tests are failing. Think it might be for real this time 😅. Could you check?

**Comment by jimezesinachi** [comment]
All of them are still working locally

Jim Ezesinachi

**Pull request merged**